packages/gatsby/src/query/better-queue-custom-store.ts

```diff
@@ -82,29 +93,33 @@ function MemoryStoreWithPriorityBuckets() {
       if (needToUpdatePriorityKeys) {
         updatePriorityKeys()
       }
-      cb()
+      cb(null)
     },
-    takeFirstN: function (n, cb) {
-      const lockId = uuid++
+    takeFirstN: function (n, cb): void {
+      const lockId = `` + uuid++
```

Comment:
⚠️ So this has changed. The interface from `better-queue` states that the `lockId` should be a string.